performance/run.js

```diff
@@ -0,0 +1,12 @@
+const { execSync } = require('child_process');
+
+if(process.platform === "darwin") {
+    execSync('docker run --rm -v "$(pwd)":/sitespeed.io sitespeedio/sitespeed.io:12.2.0 --config=config/config-chrome.json http://host.docker.internal:8080/performance/performance.html', {stdio: 'inherit', cwd: __dirname});
```

Comment:
Werkt dit ook als de demo pagina aan externe URLs moet kunnen op bamboo? Bijvoorbeeld de map.
Momenteel lijkt het er op dat er geen forward proxy wordt ingesteld. Ik denk dat we dit in een docker-compose file moeten gieten zodat http_proxy wordt meegegeven.

Reply:
Nog niet gecontroleerd, maar de proxy kan ook meegegeven worden via command-line.

Reply:
Ja, maar docker-compose krijgt die proxies al mee op bamboo. 
Voor het gewone "docker" command is dat niet het geval en moet je proberen vanuit Bamboo zelf wat variabelen te passen/hardcoden. Zal ook wel lukken hoor, maar dan moet je in dit geval vanuit uw run.js aan die variabelen geraken en doorgeven, waardoor je meerdere doorgeefluiken nodig hebt waarschijnlijk.

Reply:
Moet ik eens bekijken want we willen ook dat de testen lokaal werken.

Reply:
Er word nu met docker-compose gewerkt.